src/shared/lib/StoreUtils.ts

```diff
@@ -314,6 +316,56 @@ export async function queryOncoKbData(queryVariants: Query[],
     return oncoKbData;
 }
 
+export async function fetchCivicData(mutationData?:MobxPromise<Mutation[]>, 
+                                     uncalledMutationData?:MobxPromise<Mutation[]>) {
+    
+    const mutationDataResult = concatMutationData(mutationData, uncalledMutationData);
+    
+    let queryHugoSymbols: Array<string> = [];
+        
+    mutationDataResult.forEach(function(mutation: Mutation) {
+        queryHugoSymbols.push(mutation.gene.hugoGeneSymbol);
+    });
+
+    //For some reason, Typescript indicates that getCivicGenes can be undefined: it can't
+    let civicData: ICivicData = (await CivicAPI().getCivicGenes(queryHugoSymbols)) as ICivicData;
+
+    return civicData;
+}
+
+export async function fetchCnaCivicData(discreteCNAData:MobxPromise<DiscreteCopyNumberData[]>) {
+    
+    if (discreteCNAData.result && discreteCNAData.result.length > 0) {
+        let queryHugoSymbols: Array<string> = [];
+        
+        discreteCNAData.result.forEach(function(cna: DiscreteCopyNumberData) {
+            queryHugoSymbols.push(cna.gene.hugoGeneSymbol);
+        });
+    
+        //For some reason, Typescript indicates that getCivicGenes can be undefined: it can't
+        let civicData: ICivicData = (await CivicAPI().getCivicGenes(queryHugoSymbols)) as ICivicData;
+    
+        return civicData;
+    } else {
+        return {};
+    }
+}
+
+export async function fetchCivicVariants(civicData: ICivicData, mutationData?:MobxPromise<Mutation[]>,
+                                         uncalledMutationData?:MobxPromise<Mutation[]>) {
+
+    //For some reason, Typescript indicates that getCivicVariants can be undefined: it can't
```

Comment:
It might be because `CivicAPI` is a function but not a class. Just a guess though.